core/templates/services/assets-backend-api.service.spec.ts

```diff
@@ -194,83 +192,6 @@ describe('Assets Backend API Service', function() {
       $httpBackend.verifyNoOutstandingExpectation();
     });
 
-    it('should handler rejection on trying to process fetched file when' +
-      ' Blob throws a TypeError and BlobBuilder does not exist', function() {
-      var successHandler = jasmine.createSpy('success');
-      var failHandler = jasmine.createSpy('fail');
-
-      spyOn(window, 'Blob').and.callFake(function() {
-        throw new TypeError();
-      });
-
-      $httpBackend.expect('GET', audioRequestUrl).respond(201, 'Error');
-      expect(AssetsBackendApiService.isCached('myfile.mp3')).toBe(false);
-
-      AssetsBackendApiService.loadAudio('0', 'myfile.mp3').then(
-        successHandler, failHandler);
-      $httpBackend.flush();
-      expect(successHandler).not.toHaveBeenCalled();
-      expect(failHandler).toHaveBeenCalledWith('myfile.mp3');
-      $httpBackend.verifyNoOutstandingExpectation();
-    });
-
-    it('should handler rejection on trying to process fetched file when Blob' +
-      ' throws a TypeError and BlobBuilder does not work correctly',
-    function() {
-      var successHandler = jasmine.createSpy('success');
-      var failHandler = jasmine.createSpy('fail');
-
-      spyOn(window, 'Blob').and.callFake(function() {
-        throw new TypeError();
-      });
-
-      window.BlobBuilder = function BlobBuilder() {
-        this.blob = new Object({ blobBuilder: true });
-      };
-
-      $httpBackend.expect('GET', audioRequestUrl).respond(201, 'Error');
-      expect(AssetsBackendApiService.isCached('myfile.mp3')).toBe(false);
-
-      AssetsBackendApiService.loadAudio('0', 'myfile.mp3').then(
-        successHandler, failHandler);
-      $httpBackend.flush();
-      expect(successHandler).not.toHaveBeenCalled();
-      expect(failHandler).toHaveBeenCalledWith('myfile.mp3');
-      $httpBackend.verifyNoOutstandingExpectation();
-    });
-
-    it('should successfully process a fetch file when Blob throws a TypeError' +
```

Comment:
Make sure we have another test for the success process.

Reply:
I only removed the part where `BlobBuilder` was involved.

Reply:
Can you please check whether we still have a test that checks the successful creation of Blob out of fetched audio/images data?

[This comment wasn't published earlier]

Reply:
Here https://github.com/oppia/oppia/blob/2f8195b50e3568e2b425345da6e24f574332221e/core/templates/services/assets-backend-api.service.spec.ts#L132